business/src/main/java/org/mskcc/cbio/portal/model/DBAltCount.java

```diff
@@ -0,0 +1,80 @@
+package org.mskcc.cbio.portal.model;
+
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.io.Serializable;
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+/**
+ *
+ * @author jiaojiao
+ 
+@JsonInclude(JsonInclude.Include.NON_NULL)*/
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
+
+public class DBAltCount implements Serializable {
+    public int count;
+    public int studyID;
+    public String id;
+    public String gene;
+    public int start;
+    public int end;
+
+    public int getCount() {
```

Comment:
since all these properties are public (and will stay so), no need to have getters and setters I think